gradle.properties

```diff
@@ -13,11 +13,15 @@ org.jetbrains.gradle.plugin.idea-ext.version=0.5
 kotlin.version=1.5.31
 com.github.vlsi.vlsi-release-plugins.version=1.74
 
-calcite.version=1.28.0
-junit5.version=5.8.1
+# library version
+antlr.version=4.13.1
+calcite.version=1.36.0
+guava.version=32.1.3-jre
+immutables.version=2.10.1
+jackson.version=2.16.1
+junit.version=5.8.1
 protobuf.version=3.17.1
 slf4j.version=1.7.25
-jackson.version=2.12.4
```

Comment:
Extracted the versions of some common dependencies to make it easier to keep them in sync. Also made sure these were being used in the build.